bzt/modules/aggregator.py

```diff
@@ -264,65 +228,24 @@ def from_dict(obj):
         """
         inst = KPISet()
         for key, val in iteritems(obj):
+            if key in (inst.RESP_TIMES):
```

Comment:
Shouldn't we discard "RESP_TIMES" now, and offer some methods to get it?

Reply:
This is a `from_dict()` method used mainly for tests. It's purpose is to create a KPISet from a dictionary. For compatibility it allows passing old-style `Counter`-typed values, but this routine's job is to re-pack `Counter` into `HDRHistogram`.